aws_lambda_builders/workflows/ruby_bundler/DESIGN.md

```diff
@@ -0,0 +1,84 @@
+# Ruby - Lambda Builder
+
+## Scope
+
+For the basic case, building the dependencies for a Ruby Lambda project is very easy:
+
+```shell
+# ensure you are using Ruby 2.5, for example with rbenv or rvm
+bundle install # if no Gemfile.lock is present
+bundle install --deployment
```

Comment:
Should `bundle install` skip dev and test? `--without development test`?

Reply:
This is a fair shout. I think in an ideal world users could pass in their own options, but this would be a reasonable default.